File: kushki-android/src/test/java/com/kushkipagos/android/TestEnvironment.java
package com.kushkipagos.android;

enum TestEnvironment implements Environment {
    LOCAL("http://localhost:8888/kushki/api/v1"),
    INVALID("This causes a MalformedURLException");

    private final String url;

    TestEnvironment(String url) {
        this.url = url;
    }

    @Override
    public String getUrl() {
        return url;
    }
}

File: kushki-android/src/test/java/com/kushkipagos/android/TestEnvironment.kt
package com.kushkipagos.android

internal enum class TestEnvironment(override val url: String) : Environment {
    LOCAL("https://api-uat.kushkipagos.com/"),
    LOCAL_QA("https://api-qa.kushkipagos.com/"),
    INVALID("This causes a MalformedURLException");
}
